resources/frontend_client/inline_js/asset_loading_error.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  window.mbErrorAlert = false;
+  window.AssetErrorLoad = function (tag) {
+    console.error(
+      `Could not download asset ${tag.src} from your metabase instance. 
+This shouldn't happen normally, but can happen in certain 
+instances where your browser has cached the index.html page, or 
+there are different versions of metabase behind the same load balancer. 
+If clearing your cache doesn't resolve the issue, please ensure your
+all your deployed instances of metabase are on the same version.`,
+    );
+
+    if (!window.mbErrorAlert) {
+      alert(
```

Comment:
I would probably not display this alert and only show the error message in console. I think we _might_ not understand some cases well enough and could show this message too often